hw/ip/usbdev/doc/checklist.md

```diff
@@ -173,24 +173,24 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | Not Started |
-Documentation | [DV_DOC_COMPLETED][]                    | Not Started |
-Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     | Not Started |
-Testbench     | [ALL_INTERFACES_EXERCISED][]            | Not Started |
-Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Not Started |
-Testbench     | [SIM_TB_ENV_COMPLETED][]                | Not Started |
-Tests         | [SIM_ALL_TESTS_PASSING][]               | Not Started |
-Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | Not Started |
-Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | Not Started |
-Tests         | [SIM_FW_SIMULATED][]                    | Not Started |
-Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           | Not Started |
-Coverage      | [SIM_CODE_COVERAGE_V2][]                | Not Started |
-Coverage      | [SIM_FUNCTIONAL_COVERAGE_V2][]          | Not Started |
-Coverage      | [FPV_CODE_COVERAGE_V2][]                | Not Started |
-Coverage      | [FPV_COI_COVERAGE_V2][]                 | Not Started |
-Integration   | [PRE_VERIFIED_SUB_MODULES_V2][]         | Not Started |
-Issues        | [NO_HIGH_PRIORITY_ISSUES_PENDING][]     | Not Started |
-Issues        | [ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED][] | Not Started |
+Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | Done        |
+Documentation | [DV_DOC_COMPLETED][]                    | Done        |
+Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     | Done        |
+Testbench     | [ALL_INTERFACES_EXERCISED][]            | Done        |
+Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Done        |
+Testbench     | [SIM_TB_ENV_COMPLETED][]                | Done        |
+Tests         | [SIM_ALL_TESTS_PASSING][]               | N/A         |
+Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | N/A         |
+Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | N/A         |
+Tests         | [SIM_FW_SIMULATED][]                    | N/A         |
+Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           | Done        |
+Coverage      | [SIM_CODE_COVERAGE_V2][]                | Done        |
```

Comment:
Annoyingly, the FSM coverage numbers are still slightly under 90%. I think that will probably be sorted when the manual waiver file has been merged, but we should probably wait until the nightly run has happened with that waiver file in place.

Reply:
@alees24 and I have both run a full nightly regression of usbdev with coverage collection based on `master` + #24257.

His results:
```
|  SCORE  |  LINE   |  COND   |  TOGGLE  |   FSM   |  BRANCH  |  ASSERT  |  GROUP  |
|:-------:|:-------:|:-------:|:--------:|:-------:|:--------:|:--------:|:-------:|
| 96.76 % | 98.12 % | 95.94 % | 97.44 %  | 96.61 % | 98.34 %  | 98.17 %  | 92.67 % |
```
My results:
```
|  SCORE  |  LINE   |  COND   |  TOGGLE  |   FSM   |  BRANCH  |  ASSERT  |  GROUP  |
|:-------:|:-------:|:-------:|:--------:|:-------:|:--------:|:--------:|:-------:|
| 96.57 % | 98.11 % | 96.01 % | 97.44 %  | 94.92 % | 98.30 %  | 98.17 %  | 93.03 % |
```

I think this should suffice to tick this item off without having to wait for the next OpenTitan-wide nightly regression results.

Reply:
That's brilliant, thanks. In which case, I'm very happy with this change once the silly typo is fixed :-)